apps/livreBatch/livre_batch.py

```diff
@@ -104,7 +104,7 @@ def _build_sbatch_script(self, start, end):
         values = self.dict
         values['start'] = start
         values['end'] = end
-        values['num_frames'] = end - start + 1
+        values['num_frames'] = end - start
```

Comment:
Should have been documented like that before. I found out as well that the interval is open on the right. I'll try to document it